bigframes/core/nodes.py

```diff
@@ -284,6 +290,24 @@ def prune(self, used_cols: COLUMN_SET) -> BigFrameNode:
         return self.transform_children(lambda x: x.prune(used_cols))
 
 
+class AdditiveNode:
+    """Definition of additive - if you drop added_fields, you end up with the descendent."""
```

Comment:
A picture might help :-) 

```suggestion
    """Definition of additive - if you drop added_fields, you end up with the descendent.

    .. code-block:: text

        AdditiveNode (fields: a, b, c; added_fields: c)
            |
            |  additive_base
            V
        BigFrameNode (fields: a, b)

    """
```

See https://stackoverflow.com/a/50956831/101923 for creating a plain text block